src/server.h

```diff
@@ -518,8 +518,8 @@ typedef enum {
 /* Anti-warning macro... */
 #define UNUSED(V) ((void) V)
 
-#define ZSKIPLIST_MAXLEVEL 32 /* Should be enough for 2^64 elements */
-#define ZSKIPLIST_P 0.25      /* Skiplist P = 1/4 */
+#define ZSKIPLIST_MAXLEVEL 32     /* Should be enough for 2^64 elements */
```

Comment:
```suggestion
#define ZSKIPLIST_MAXLEVEL 32 /* Should be enough for 2^64 elements */
```
no need to change it?